README.md

```diff
@@ -1,2 +1,224 @@
-# CircleCI-HockeyApp-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: CircleCI, HockeyApp, Appium and ReactNative
+# Run Kobiton Automation Test on CircleCI
+
+This document will guide you how to use CircleCI to execute automation test on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents
+- [A. Preparation](#a-preparation)
+    - [1. Prerequisites](#1-prerequisites)
+    - [2. Getting Kobiton Username and API key](#2-getting-kobiton-username-and-api-key)
+    - [3. Samples](#3-samples)
+- [B. Setup](#b-setup)
+    - [Purposes](#purposes)
+    - [1. Setting Kobiton Username and API Key](#1-setting-kobiton-username-and-api-key)
+    - [2. Getting Kobiton device desired capabilities](#2-getting-kobiton-device-desired-capabilities)
+    - [3. Setting desired capabilities](#3-setting-desired-capabilities)
+    - [4. CircleCI configuration file](#4-circleci-configuration-file)
+    - [5. Configuring HockeyApp WebHook listener](#5-configuring-hockeyapp-webhook-listener)
+- [C. Execution](#c-execution)
+    - [1. Executing the automation test](#1-executing-the-automation-test)
+    - [2. Fetching test session data using Kobiton REST API](#2-fetching-test-session-data-using-kobiton-rest-api)
+- [D. Feedback](#d-feedback)
+
+## A. Preparation
+
+### 1. Prerequisites
+- A server for receiving WebHooks.
+    - Can be virtual, bare-metal or container, etc.
```

Comment:
I don't think this line is necessary unless there are exception(s) in configuring the server.